products/firefox_android/testing.md

```diff
@@ -0,0 +1,34 @@
+# Testing Firefox for Android
+
+## Testing on single-locale builds
+
+Once you have localized [all the strings needed to complete the Firefox for Android project](/products/firefox_android/localize_android.md), and your language is now up on single-locale builds, you should start testing! This is the moment to engage as much people as possible: have them use and check-out these builds regularly. The more eyes on them, the better.
```

Comment:
I'm not sure they would be fixed once merged, moving to relative links just in case.